src/scripts/shell/star-rating/star-rating.html

```diff
@@ -0,0 +1,11 @@
+<span ng-if="$ctrl.isReadOnly">
+  <i ng-repeat="star in $ctrl.stars" class="fa fa-star read-only"
+   ng-class="{filled: star.filled}"></i>
+</span>
+<span ng-if="!$ctrl.isReadOnly">
```

Comment:
Originally had it that way (does look cleaner), figured this way the events would never even be bound. I'll swap it back